src/modals/PodEdit.tsx

```diff
@@ -141,7 +142,7 @@ function PodEditComponent({ open, closeModal, modalParams }: Props) {
       closeTimeoutMS={100}
       style={style}
     >
-      <PodEditContainer>
+      <PodEditContainer data-name="pod-edit-container">
```

Comment:
We use camelCase for data-name

Reply:
Thanks for letting me know. The attribute is updated.